lib/src/theme/colors.dart

```diff
@@ -36,6 +37,14 @@ class DSFRColors extends ThemeExtension<DSFRColors> {
   final Color badgeWarning;
   final Color badgeNews;
   final Color banner;
+  final Color radioActive;
+  final Color frLabel;
+  final Color frFieldsetDisabled;
+  final Color frHintText;
+  final Color g200;
+  final Color g600;
+  final Color g700;
+  final Color g800;
```

Comment:
don't know if colors should be named like that? It looks more like a name for the Palette class. But yeah a lot of color are duplicated so maybe we need to have some common color in the DSFRColors class. Is it really a problem if we specify what the color is used for even if it leads to duplicates?

Reply:
Actually those names were not chosen by me, it's the real name of those color in the design system if you take a look at the variables. It's not about having duplicate colors but really having the correct color associated with the correct variables to ensure flexibility of the color scheme.
Now, I agree the name are not the most explicit so we could consider adding comments to describe some example usage for those colors.

Reply:
Yeah, but what if we keep the name g700 in colorPalette, and then in the DSFRColors class we name it something like `radioBackground` ? This is more human friendly and maintainable, but can lead to some dupplicate e.g some DSFRColors properties could be the same, but with different name

Reply:
Actually, it wouldn't be more maintainable because of the color scheme, if at some points the design system add a variant or change the value of g700 then we would have to remember all the transitive variables we've made to replace them.
The best alternative I could give you would be to create a getter on g700 with a more human friendly name.

Reply:
If we put g700 in palette.dart and then the design system change the value of g700, we will just update the variable in palette.dart and every properties that use g700 in colors.dart will be automatically updated. Though I'm ok with the getter idea

Reply:
No because g700 is different in light and dark mode

Reply:
Ok I see that's just very bad naming